pybind/README.md

```
@@ -0,0 +1,42 @@
+# Running EmotiBit Brainflow SpO2 Algorithm using `pybind`
+## Requirements
+- Python
+- Python virtual environment (refer to the Emotibit Plugins [repo](https://github.com/EmotiBit/EmotiBit_Plugins/blob/main/README_py.md#Creating-python-env-with-pybind11) for specific instructions on how to setup)
+- CMake
+- [EmotiBit ArduinoFilters](https://github.com/EmotiBit/EmotiBit_ArduinoFilters)
+  - The EmotiBit ArduinoFilter directory should be on the same level as this repository.
+  - **Option 1:** Library Manager 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)